langchain4j-core/src/main/java/dev/langchain4j/agent/tool/Tool.java

```diff
@@ -36,4 +36,15 @@
      * @return description of the tool.
      */
     String[] value() default "";
+
+    /**
+     * Indicates whether the tool's String return value should be returned directly to the user.
+     * Only applicable if used within an AiService and if the return type is 'Result<String>'
+     * If true, the result will be returned directly without further LLM processing.
+     * If false or if return type is not Result<String>, normal processing continues.
+     * If true, the tool cannot be chained with other tools as the result will not be returned to the llm.
+     *
+     * @return whether to return the result directly
+     */
+    boolean directReturn() default false;
```

Comment:
I am wondering what is the more appropriate name: `directReturn` or `returnDirectly` or maybe something else?
Perhaps something more descriptive like `returnResultDirectly`? Or something that highlights that LLM is not involved in the second turn? WDYT?

Reply:
Good point, as usual naming are the hardest part :) Yes, I would like to make clear that the tool result is no further manipulated by the LLM and I'm not sure that `direct` is the best adjective to convey this message. What about `raw`? Something like `returnRawResult`?